app/ide-desktop/lib/content/src/style.css

```diff
@@ -1,81 +1,71 @@
 /* Fonts */
 
 @font-face {
-  font-family: "Causten";
-  src: url("/assets/fonts/CaustenThin/font.woff2") format("woff2"),
-    url("/assets/fonts/CaustenThin/font.woff") format("woff");
-  font-style: normal;
+  font-family: "M PLUS 1";
+  src: url("/assets/MPLUS1[wght].ttf") format("truetype");
   font-weight: 100;
   font-display: block;
 }
 
 @font-face {
-  font-family: "Causten";
-  src: url("/assets/fonts/CaustenExtraLight/font.woff2") format("woff2"),
-    url("/assets/fonts/CaustenExtraLight/font.woff") format("woff");
+  font-family: "M PLUS 1";
+  src: url("/assets/MPLUS1[wght].ttf") format("truetype");
   font-style: normal;
   font-weight: 200;
```

Comment:
It's quite problematic to re-use existing script because all the build logic is in the ci-build repository now. I should've linked the related PR https://github.com/enso-org/ci-build/pull/9